src/tests/system/mhc_2.yaml

```diff
@@ -0,0 +1,65 @@
+domains:
```

Comment:
Why did you add a new `mhc.yaml` file? Is it actually needed?

Reply:
No, not needed.   Was accidentally added.